install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -674,10 +674,10 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         command:
           - "/install-plugin.sh"
+        {{- with .Values.cni.resources }}
```

Comment:
One minor comment just for awareness: Because this is dereferencing a new option, we cannot consider backporting this change. It would need to do a null check first if we want to port it to an existing release. As such this PR would first arrive in an upcoming `v1.15-pre.*` snapshot and of course in the final v1.15.x releases.